wiki/en/Running-a-Server.md

```diff
@@ -19,87 +19,67 @@ permalink: "/wiki/Running-a-Server"
+Many people attribute problems to the Server that are in fact problems with the _client_. Much depends on the clients' hard/software setup, the networks that _they_ are on, and whether they are sticking to [Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
 
-**_The capability of the server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**
+**_The capability of the Server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**
 
 If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
 
-Once any issues with musicians have been solved in this way, you can then investigate hosting your own server either at home or on a cloud host such as Amazon, which may result in better latency than servers run at home. For example, [see this guide](https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/) for using AWS Lightsail, by Jamulus user [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a) (_Facebook_)
+Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home. For example, [see this guide](https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/) for using AWS Lightsail, by Jamulus user [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a) (_Facebook_)
 
 ### Bandwidth – do you have enough?
 
 Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at different quality settings later in this document.
 
-
 ### In general
 
 - Consider using a cloud host to get better ping times if you're having problems
 
-- Any server should have at least 1.6GHz CPU frequency and 1GB RAM
-
-- Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
+- Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
```

Comment:
That's not very useful without saying what capacity it would support - how many channels should someone expect to support with that spec?  10? 5? 150?

Reply:
The whole debate about sizing is for another day I think. That line has been there for at least three years I think.